src/Surfnet/StepupMiddleware/ApiBundle/Identity/Repository/RaListingRepository.php

```diff
@@ -237,4 +237,9 @@ public function removeByIdentityIdAndInstitution(IdentityId $identityId, Institu
             ->getQuery()
             ->execute();
     }
+
+    public function contains(IdentityId $identityId): bool
+    {
+        return count(parent::findBy(['identityId' => (string)$identityId])) > 0;
```

Comment:
Interestingly `parent::findBy` is used instead of `$this->findBy` :thinking: 
Do you know if there's a reason? Or just copy/paste?